community/iptables/build

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+# give option to easily enable nftables by passing USE="--with-nftables" to kiss
+[ -z "$USE" ] && USE="--disable-nftables"
+
+autoreconf -i
+
+./configure --prefix=/usr \
+            --sbindir=/usr/sbin \
+            --enable-libipq --with-xtlibdir=/lib/xtables "$USE"
+
+make
+
```

Comment:
Remove this blank line.

Reply:
look again, i was already fixing before you commented

Reply:
Apologies. :+1:

Reply:
Try again, i'm confused which blank line is troublesome

Reply:
The blank line between `make` and `make install`.

Reply:
okay got it  thanks, was worried because Emacs throws an extra line EOF unless I change a setting.